Documentation/ceph-upgrade.md

```diff
@@ -472,6 +507,53 @@ periodically. If this is the case, it is easiest to `kubectl edit` the Operator
 modify everything needed at once. You can switch between using Rook-Ceph's default CSI images and
 custom CSI images as you wish.
 
+### Update snapshot CRD from Alpha to Beta
+
+CSI snapshot has a breaking change, we need to delete the snapshotterAlpha CRD
+created by external-snapshotter sidecar container.
+
+As we are updating the snapshot resources from `Alpha` to `Beta` we need to
+delete the old `alphav1` snapshot CRD created by external-snapshotter sidecar container
+
+Check if we have any `v1alpha1` CRD created in our kubernetes cluster
+
+```bash
+[$]kubectl get crd volumesnapshotclasses.snapshot.storage.k8s.io -o yaml |grep v1alpha1
+  - name: v1alpha1
+  - v1alpha1
+[$]kubectl get crd volumesnapshotcontents.snapshot.storage.k8s.io -o yaml |grep v1alpha1
+  - name: v1alpha1
+  - v1alpha1
+[$]kubectl get crd volumesnapshots.snapshot.storage.k8s.io -o yaml |grep v1alpha1
+  - name: v1alpha1
+  - v1alpha1
+```
+
+As we have `v1alpha1` CRD created in our kubernetes cluster,we need to delete
+the Alpha CRD
+
+```bash
+[$]kubectl delete crd volumesnapshotclasses.snapshot.storage.k8s.io
+customresourcedefinition.apiextensions.k8s.io "volumesnapshotclasses.snapshot.storage.k8s.io" deleted
+[$]kubectl delete crd volumesnapshotcontents.snapshot.storage.k8s.io
+customresourcedefinition.apiextensions.k8s.io "volumesnapshotcontents.snapshot.storage.k8s.io" deleted
+[$]kubectl delete crd volumesnapshots.snapshot.storage.k8s.io
+customresourcedefinition.apiextensions.k8s.io "volumesnapshots.snapshot.storage.k8s.io" deleted
+```
+
+**Note**: Its kubernetes distributor responsibility to install new snapshot
+controller and snapshot beta CRD. more info can be found
+[here](https://github.com/kubernetes-csi/external-snapshotter/tree/master#usage)
+
+### Update the RBAC required for external-snapshotter
```

Comment:
Actually we can remove this section. There will be a section below that already applies the `upgrade-from-v1.3-apply.yaml` for all the RBAC updates in the release.

Reply:
`upgrade-from-v1.3-apply.yaml` is not mentioned in this doc yet, do I need to remove this one, or once we add more RBAC we can remove from here and move it to top section?

Reply:
This is fine for now, we can merge it to another section when the upgrade guide is updated immediately before the 1.4 release